micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/citizen/ComplaintDetails.js

```
@@ -0,0 +1,186 @@
+    let timeline = data?.timeline;
+    timeline && timeline[0].timeLineActions?.filter((e) => e === "COMMENT").length ? setDisableComment(false) : setDisableComment(true);
+    if (timeline) {
+      const actionByCitizenOnComplaintCreation = timeline.find((e) => e?.performedAction === "APPLY");
+      const { thumbnailsToShow } = actionByCitizenOnComplaintCreation;
+      setImageToShowBelowComplaintDetails(thumbnailsToShow);
+    }
+  };
+
+  const submitComment = async () => {
+    let detailsToSend = { ...complaintDetails };
+    delete detailsToSend.audit;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)